compiler/testData/codegen/bytecodeText/nullCheckOptimization/deterministicNotNullChecks.kt

```diff
@@ -1,8 +1,4 @@
 // FILE: test/CallableDescriptor.java
-// IGNORE_BACKEND_FIR: JVM_IR
-// Here FIR adds implicit NOT_NULL cast for `origin`, resulting in an assertion being added,
-// which is the correct (yet mismatching) behavior, according to https://youtrack.jetbrains.com/issue/KT-35656
-
```

Comment:
Duh, I'm removing my own comments. 🤦  Well, on the bright side, at least now we match with JVM_IR behavior.